apps/backend/src/mcp/tools/device_info.py

```
@@ -203,6 +200,775 @@ def _parse_smart_output(smart_output: str, drive_path: str) -> Dict[str, Any]:
     return smart_data
 
 
+async def _analyze_connectivity(
+    device: str, ssh_client, connection_info: SSHConnectionInfo
+) -> Dict[str, Any]:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)